04_HowTos/SPARCL/Plot_SPARCL_Spectra_with_Jdaviz.ipynb

```diff
@@ -0,0 +1,474 @@
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client = SparclClient()\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0de20865-2ad4-4235-9a26-653f05510f30",
+   "metadata": {},
+   "source": [
+    "#### View the full list of fields for the DESI EDR data set that can be obtained from SPARCL"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "9e288542-73aa-45cf-9f69-668bd3208d70",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-10-10T19:54:18.420680Z",
+     "iopub.status.busy": "2023-10-10T19:54:18.420212Z",
+     "iopub.status.idle": "2023-10-10T19:54:18.427555Z",
+     "shell.execute_reply": "2023-10-10T19:54:18.426376Z",
+     "shell.execute_reply.started": "2023-10-10T19:54:18.420628Z"
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['bgs_target', 'chi2', 'cmx_target', 'coadd_fiberstatus', 'coadd_numexp', 'coadd_numnight', 'coadd_numtile', 'coeff', 'data_release', 'datasetgroup', 'dateobs', 'dateobs_center', 'dec', 'deltachi2', 'desi_target', 'exptime', 'fa_target', 'fa_type', 'firstnight', 'flux', 'healpix', 'instrument', 'ivar', 'lastnight', 'mask', 'mean_delta_x', 'mean_delta_y', 'mean_fiber_dec', 'mean_fiber_ra', 'mean_mjd', 'mean_psf_to_fiber_specflux', 'model', 'mws_target', 'ncoeff', 'npixels', 'numobs_init', 'objtype', 'obsconditions', 'plate_dec', 'plate_ra', 'pmdec', 'pmra', 'priority_init', 'program', 'ra', 'redshift', 'redshift_err', 'redshift_warning', 'ref_epoch', 'rms_delta_x', 'rms_delta_y', 'scnd_target', 'site', 'sparcl_id', 'specid', 'specprimary', 'spectype', 'spgrpval', 'std_fiber_dec', 'std_fiber_ra', 'subpriority', 'subtype', 'survey', 'sv1_bgs_target', 'sv1_desi_target', 'sv1_mws_target', 'sv1_scnd_target', 'sv2_bgs_target', 'sv2_desi_target', 'sv2_mws_target', 'sv2_scnd_target', 'sv3_bgs_target', 'sv3_desi_target', 'sv3_mws_target', 'sv3_scnd_target', 'sv_nspec', 'sv_primary', 'target_dec', 'target_ra', 'targetid', 'telescope', 'tsnr2_bgs', 'tsnr2_elg', 'tsnr2_gpbbackup', 'tsnr2_gpbbright', 'tsnr2_gpbdark', 'tsnr2_lrg', 'tsnr2_lya', 'tsnr2_qso', 'wave_sigma', 'wavelength', 'wavemax', 'wavemin', 'zcat_nspec']\n"
+     ]
+    }
+   ],
+   "source": [
+    "desi_fields = client.get_all_fields(dataset_list=['DESI-EDR'])\n",
+    "print(desi_fields)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f2b74af-e9ea-4845-bb6d-38ae1425067a",
+   "metadata": {},
+   "source": [
+    "<a class=\"anchor\" id=\"searchsparcl\"></a>\n",
+    "## Search for spectra in SPARCL\n",
+    "Using the `client.find()` method, we obtain the following fields from the DESI EDR data set using SPARCL:  \n",
+    "- `sparcl_id` : Universally Unique Identifier for spectrum in SPARCL\n",
```

Comment:
lower case 'u'?

Reply:
"Universally Unique Identifier" is a genuine proper noun.